core_brain/markout.py

```
@@ -109,6 +268,18 @@ def sample_pending_markouts(
         mid = mids.get(leg) if leg else None
+
+        # The windowed reference and the peer baseline, when the tape can be
+        # read. Best-effort by design: a tape that will not answer leaves these
+        # NULL and the raw mid-based markout is unaffected, which is the same
+        # contract the rest of this sampler runs under.
+        if mid is not None and trades_fn is not None:
+            try:
+                _record_reference(registry, row, m_id, h_idx, mid, row_token,
+                                  trades_fn, now, horizons)
+            except Exception:
+                pass
```

Ruff (0.16.2)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)